src/itest/java/uk/gov/companieshouse/exemptions/delta/Configuration.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.exemptions.delta;
+
+import io.cucumber.spring.CucumberContextConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+
+@CucumberContextConfiguration
+@AutoConfigureMockMvc
+@SpringBootTest(classes = Application.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+@EmbeddedKafka(
+        topics = {"company-exemptions-delta", "company-exemptions-delta-company-exemptions-delta-consumer-retry",
+                "company-exemptions-delta-company-exemptions-delta-consumer-error", "company-exemptions-delta-company-exemptions-delta-consumer-invalid"},
+        controlledShutdown = true,
+        partitions = 1
+)
+@TestPropertySource(locations = "classpath:application-integration_consumer_upsert.yml")
+@Import(TestConfig.class)
+@ActiveProfiles("integration_consumer_upsert")
```

Comment:
Are we actually using Spring Profiles? If not then this can go

Reply:
Gone